src/App.tsx

```diff
@@ -59,6 +61,29 @@ function App() {
     );
   }
 
+  function onDragEnd(result: DropResult) {
+    const { destination, source, draggableId } = result;
+
+    if (!destination) {
+      return;
+    }
+
+    if (
+      destination.droppableId === source.droppableId &&
+      destination.index === source.index
+    ) {
+      return;
+    }
+
+    const newTodoItems = Array.from(todoItems);
+    const tmpItem = todoItems[source.index];
+
+    newTodoItems.splice(source.index, 1);
```

Comment:
I would like to see comments here. For example: `// delete dragged item from the original array`